pipelines/nextflow/modules/genome_metadata/datasets_metadata.nf

```diff
@@ -0,0 +1,40 @@
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+process DATASETS_METADATA {
+    tag "$accession"
+    label 'local'
+    label 'cached'
+
```

Comment:
Shall we add the container here? Since `NXF_SINGULARITY_CACHEDIR` is pointing to were our current container is, it should work for us. We can wait until the next containers meeting to see how we release it to the world after that. I assume `nextflow.config` will need to be also updated to enable singularity.

Reply:
When we agree on the container yes